app/src/main/res/values/styles.xml

```diff
@@ -169,6 +169,14 @@
         <item name="android:textStyle">normal</item>
     </style>
 
+    <style name="H3.Radio">
```

Comment:
The styles called `H...` indicate a "heading", and are not really intended for things that are not headings. Can we make this style based on `P` instead of `H`?

Reply:
It makes sense to me. Done.